src/catalogue/kubernetes/catalogue-dep.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         - name: wallet
           mountPath: /wallet
       - name: init-service-database
-        image: iad.ocir.io/oracle/ateam/mushop-dbtools:1.0.1
+        image: iad.ocir.io/oracle/ateam/mushop-dbtools:21
```

Comment:
So... I see where the `21` is used to represent the client version, and the `dbtools` is updated to reflect that. But, is that necessarily what we want to do? Might there be a case where "MuShop" would want to rev `dbtools` to 21.0.1 to update logic|scripts|whatever thus making it out of sync with the lexical client version? My feeling is that the dbtools version should just be a major bump to `2.0.0` in this case.

Reply:
dbtools just have the unmodified OracleDB tools, there are no scripts. The Instant Client changed the strategy to work with major version support, that's why skip 20, and 21 will last multiple version of the client and db.

If the InstantClient receives a major change, such as new authentication method, will change the major (22+).

So, the main binaries will still following the InstantClient/DB server major version strategy. And all changes can be tracked with minor.patch.

Reply:
> dbtools just have the unmodified OracleDB tools, there are no scripts. The Instant Client changed the strategy to work with major version support, that's why skip 20, and 21 will last multiple version of the client and db.

☝️ sort of. It still has this Dockerfile with installs and whatnot that we might need to change periodically. Theoretically the instant client can have its own semver that could differ. I wouldn't want there to be confusion as a result

Reply:
Oracle should provide that image, it's generic as is. Should only change if the InstantClient changes. There's no other reasons to make updates. We were using version 1.0.1 since the beginning and was using the old 19.3. The 1.0.2 was never used because was updated to 19.5 and break changes.

One of the reasons that the InstantClient changed the strategy, and started to use major version instead of major.minor, was exactly to avoid that issues, as some patches are necessary.

Including the version on the image tag, will also help when creating the volumes.

Some features are only available on the new DB client, and for the developer, `dbtools:2.0.0` will not tell anything, and the developer will be unaware why is not working. Will try to look at the dbtools:2.0.0 description on the docker repo, and will not see anything. Will just bring unnecessary time lost trying to debugging.

This is also some best practices that are being applied on other client tools.